source/user-manual/capabilities/file-integrity/fim-fields-rule-mapping.rst

```diff
@@ -0,0 +1,145 @@
+.. Copyright (C) 2021 Wazuh, Inc.
+
+Fim fields rule mapping
+=======================
+
+This guide aims to help to understand how FIM fields can be mapped into rules.
+
+
+FIM - Alerts : fields correspondence
+------------------------------------
+
+The following table establishes a correspondence between the syscheck fields and their homologs in the alert generation.
+
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  **Syscheck field**  |  **Alert field**            | **Field description**                                           |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  file                |  path                       |  File path in the current alert                                 |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  size                |  size_after                 |  File size in the current alert                                 |
```

Comment:
```suggestion
|  size                |  size_after                 |  File size in the current event                                 |
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3913/commits/6f538ba0f29e9eb74ddfc9e363f4de470e21bb84)